trainingMaquetado/Maquetado.css

```diff
@@ -0,0 +1,49 @@
+hr {
+	background-color: #F5C85F;
+	width: 4px;
+	height: 300px;
+	margin: 60px;
+	border: none;
+  border-radius: 60px;
+  flex-shrink: 0; 
+  background: radial-gradient(#F5C85F 40%, white 60%);
+}
+
+.headline {
+	display: flex;
+	background-color: #F5C85F;
+	height: 4px;
+}
+
+
+.upper-container {	
+	display: flex;	
+	align-items: center;
+	justify-content: center;
+	height: 374px;
```

Comment:
don't set a fixed height, let it adapt to the content it has inside. use `padding` to separate this element from its content if needed.